salesagility/SuiteCRM
# fixed #6050 No submit button while editing imported email

fix for #6050 - No submit button while editing imported email

## Description
Fix will use the correct folder for import.

## Motivation and Context
To resolve the issue listed in #6050 

## How To Test This
1. Set up group email account
2. Import a selected email
3. Set up a personal email account
4. Import a selected email
5. Switch the folders and see that the imported emails flag is changing and also the listed issue is fixed

## Types of changes


### Final checklist
